frontend/src/components/TemplateSummaryModal.tsx

```diff
@@ -0,0 +1,198 @@
+                        ></ComposableTable>
+                    </div>
+                </ExpandableSection>
+            )}
+            {clusterCurator.spec?.upgrade && (
+                <ExpandableSection
+                    onToggle={() => setUpgradeExpandable(!isUpgradeExpandableOpen)}
+                    isExpanded={isUpgradeExpandableOpen}
+                    className={classes.expandableSection}
+                    toggleText={t('Upgrade')}
+                    isIndented
+                >
+                    <ComposableTable
+                        title={t('template.preUpgrade.label')}
+                        curatorJobs={
+                            clusterCurator.spec.upgrade.prehook?.map(
+                                (job: ClusterCuratorAnsibleJob) => job.name
+                            ) as string[]
+                        }
+                    ></ComposableTable>
+                    <div className={classes.expandableSection}>
+                        <ComposableTable
+                            title={t('template.postUpgrade.label')}
+                            curatorJobs={
+                                clusterCurator.spec.upgrade.posthook?.map(
+                                    (job: ClusterCuratorAnsibleJob) => job.name
+                                ) as string[]
+                            }
+                        ></ComposableTable>
+                    </div>
+                </ExpandableSection>
+            )}
+            {/* {curator.spec?.scale && <ExpandableSection></ExpandableSection>}
+            {curator.spec?.destroy && <ExpandableSection></ExpandableSection>} */}
+        </div>
+    )
+}
+
+export default function TemplateSummaryModal(props: ITemplateSummaryModalProps) {
+    const { curatorTemplate, isOpen, close } = props
+    const { t } = useTranslation()
+    return (
+        <AcmModal
+            title={t('Automation template for {{curatorName}}', { curatorName: curatorTemplate.metadata.name })}
+            variant={ModalVariant.medium}
+            isOpen={isOpen}
+            onClose={close}
+        >
+            <TemplateSummaryExpandable clusterCurator={curatorTemplate}></TemplateSummaryExpandable>
+        </AcmModal>
+    )
+}
+
+function ComposableTable(props: { title: string; curatorJobs?: string[] }) {
+    const { curatorJobs, title } = props
+
+    return (
+        <TableComposable aria-label="Simple table" variant={'compact'}>
+            {/* <Caption>Simple table using composable components</Caption> */}
+            <Thead>
+                <Tr>
+                    <Th style={{ padding: '0px 0px 8px 0px' }}>{title}</Th>
+                </Tr>
+            </Thead>
+            {curatorJobs && (
+                <Tbody>
+                    {curatorJobs.length > 0 ? (
+                        curatorJobs.map((item) => (
+                            <Tr key={item}>
+                                <Td style={{ padding: '8px 0px' }} dataLabel={item}>
```

Comment:
Just an FYI, this line and the <Th> above it is what I was talking about. At first I was apply a useStyle hook here. Did you make an alteration on it by any chance? It looks like we can't use my useStyle hook with them anyway, because their className props are being overwritten in the component...So I assume the change back was intentional, or maybe git has become sentient haha. Just wanted to point out what I observed in the earlier comment on the PR thread.